src/ruby/yast/logger.rb

```diff
@@ -1,6 +1,10 @@
 require "yastx"
 require "yast/builtins"
 
+# FIXME: a workaround for https://github.com/yast/yast-ruby-bindings/issues/133
+require "enc/encdb.so"
+require "enc/trans/transdb.so"
```

Comment:
Interesting, I put

```c
  rb_require("enc/encdb.so");
  rb_require("enc/trans/transdb.so");
```

there and it did not help and raise the `code converter not found (ASCII-8BIT to UTF-8)`... :question:
(I tried it also without the `.so` suffix, no change...)